templates/contacts/contact_invite_participants.haml

```diff
@@ -439,49 +450,73 @@
 
 
     /* Invite functional for selected opt-in flow */
+    var message = $("#id_message");
     var saveBtn = $("#id_save_flow_btn");
     var flowSelect = $("#id_opt_in_flow");
     var flowEditLink = $("#id_open_in_flow_editor");
     var inviteLinks = $('a.invite');
-    inviteLinks.hide();
+
+    message.hide();
+    inviteLinks.attr("data-available", false);
 
     function initInviteLinks() {
       let flowID = JSON.parse(flowSelect.val()).id;
       inviteLinks.attr('href', flowSelect.attr('data-url-template').replace('0', flowID));
-      inviteLinks.show();
+      inviteLinks.attr("data-available", true);
     }
 
     function processInvite(url, params) {
       let token = $('input[name=csrfmiddlewaretoken]').val();
       $.ajax({
         url: url,
         type: "POST",
+        contentType: 'application/x-www-form-urlencoded',
         data: {
```

Comment:
I've tested and the flow is not being started to the contact, maybe it's missing the flow parameter here.

Reply:
I've changed logic according to 1.0 Now it sends requests to the current page through `GET` with param `contact_uuid`.